docs/codeql/codeql-language-guides/customizing-library-models-for-javascript.rst

```diff
@@ -166,6 +166,17 @@ We can recognize this using the following extension:
 
 This works in this example because the **connection** parameter has a type annotation that matches what the model is looking for.
 
+Note that there is a significant difference between the following two rows:
+
+.. code-block:: yaml
+
+        data:
+        - ["mysql.Connection", "", ...]
+        - ["mysql", "Member[Connection]", ...]
+
+The first row matches instances of **mysql.Connection** which are objects that encapsulate a MySQL connection.
```

Comment:
```suggestion
The first row matches instances of **mysql.Connection**, which are objects that encapsulate a MySQL connection.
```

Super minor suggestion, but also using this to say if there is a simpler word than "encapsulate" I'd recommend it

Reply:
"encapsulate" is a commonly used term in programming literature, I think it should be fine here

Reply:
Cool 👍